client/src/components/selects/addExtensionSelect.tsx

```diff
@@ -136,7 +136,8 @@ const AddExtensionSelect = ({
           canEscapeKeyClose: true,
           lazy: true,
           position: Position.RIGHT_TOP,
-          usePortal: true
+          usePortal: true,
+          interactionKind: PopoverInteractionKind.HOVER
```

Comment:
Won't that display the popover whenever the user hovers onto the fhirResourceTree?

Reply:
I think these props are applied on the popover directly, not to the resourceTree